nixos/modules/module-list.nix

```diff
@@ -608,6 +608,8 @@
   ./services/networking/atftpd.nix
   ./services/networking/avahi-daemon.nix
   ./services/networking/babeld.nix
+  ./services/networking/bee.nix
+  ./services/networking/bee-clef.nix
```

Comment:
Adding a [nixos test](https://nixos.org/manual/nixos/stable/index.html#sec-nixos-tests) for these modules would be nice if possible.

Reply:
i skimmed the docs, this sounds great! but i'm new to nix, and i would like to first have the package merged. 

bee itself is still in flux, so i just noted this as a TODO for myself.